# Restrictions on using Execut and Eval on the server

Restrictions on using Execut and Eval on the server

## Noncompliant Code Example

Execut algorithm;

## Compliant Solution

SetSafeMode(True);
Execut algorithm;

 ## See

- [Restrictions on using Execut and Eval on the server](https://its.1c.ru/db/v8std#content:770:hdoc)